development-updates.md

```diff
@@ -73,6 +74,7 @@ With the gained insight into the protocol, the following weeks serve as a deep d
 | -------------------------------------------------------------- | --------------------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | [Abims](https://github.com/mr-abims)                           |                                                                                               |                                                                                               |                                                                                                                                                         |
 | [Ademola](https://github.com/0xcrust)                          |                                                                                               |                                                                                               |                                                                                                                                                         |
+| [AdedamolaXL](https://github.com/AdedamolaXL)                           | [Update 3](https://github.com/AdedamolaXL/ephemery-resources/blob/b906c5710a369ed096791233811f483753e04f68/specs.md)                                                                                          |                                                                                         |                                                                                                               |
```

Comment:
please make sure to link an actual update, not a PR

Reply:
yes, i made a mistake there, i will correct it.

about the pr, can you take a look at [ephemery](https://github.com/ephemery-testnet/ephemery-resources/pulls) pr and evaluate if they are good additions to the spec.md file?

if it is I can make a summary of it on hackmd or link update3 to the specs.md file in my own fork.

*i also don't know how my row was deleted, it was there last week as far as I can remember*

nevertheless I will try and link update3 correctly, am guessing update 0 & 1 is fine ?

I referenced my plans in [notes.md](https://github.com/AdedamolaXL/cohort-four/blob/master/notes/adedamolaxl.md), i will make it even more explicit in the coming days.